playbooks/openstack/openshift-cluster/uninstall.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Looking at this, I wonder whether this shouldn't be called unprovision instead. Having this alongside `install.yml` implies to me it'll leave the OpenStack resources untouched and it just removes the OpenShift installation.

Do you agree? I don't like the "unprovision" word, but the install/uninstall asymmetry worries me. Maybe something like "remove-cluster"? Might be worth seeing what the say the AWS folks are doing.

Reply:
It looks like the AWS folks are also using uninstall.yml. The asymmetry worries me too a bit; I guess if we were to be fully symmetrical, we'd call it unprovision_uninstall.yml? Let me know what you think!

Reply:
Ah okay. In that case let's keep it as uninstall. I don't like it, but it's more important to be consistent with the other cloud providers. Maybe we can rename it everywhere later.